akka-persistence/src/main/scala/akka/persistence/SnapshotProtocol.scala

```diff
@@ -24,6 +24,7 @@ final case class SnapshotMetadata(persistenceId: String, sequenceNr: Long, times
  */
 @SerialVersionUID(1L)
 final case class SaveSnapshotSuccess(metadata: SnapshotMetadata)
+  extends SnapshotProtocol.Response // Internal snapshot acknowledgement.
```

Comment:
* These messages are user API, so not internal - remove the comment please (internal snapshot messages are listed in `private[persistence] object SnapshotProtocol {`)
* Was thinking if `Response` is the best type here, but I guess it is, even if the user does not *directly* send a request (he does indirectly, by calling a `saveSnapshot` method).
* What about `SnapshotOffer`, shouldn't we also mark it for completeness? It would be a `Message` then IMO.